Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -121,25 +121,25 @@
     <value>Request to Amazon Elastic Container Registry failed prematurely. This is often caused when the target repository does not exist in the registry.</value>
   </data>
   <data name="BlobUploadFailed" xml:space="preserve">
-    <value>Failed to upload blob to {0}; received {1} with detail {2}</value>
+    <value>Failed to upload blob to {0}; received {1} with detail {2}.</value>
   </data>
   <data name="CouldntDeserializeJsonToken" xml:space="preserve">
-    <value>Could not deserialize token from JSON</value>
+    <value>Could not deserialize token from JSON.</value>
   </data>
   <data name="DockerInfoFailed" xml:space="preserve">
-    <value>Failed to get docker info({0})\n{1}</value>
+    <value>Failed to get docker info({0})\n{1}\n{2}.</value>
   </data>
   <data name="DockerProcessCreationFailed" xml:space="preserve">
-    <value>Failed creating docker process</value>
+    <value>Failed creating docker process.</value>
   </data>
   <data name="DontKnowHowToPullImages" xml:space="preserve">
-    <value>Don't know how to pull images from local daemons at the moment</value>
+    <value>Don't know how to pull images from local daemons at the moment.</value>
   </data>
   <data name="FailedRetrievingCredentials" xml:space="preserve">
-    <value>Failed retrieving credentials for "{0}": {1}</value>
+    <value>Failed retrieving credentials for "{0}": {1}.</value>
   </data>
   <data name="ImageLoadFailed" xml:space="preserve">
-    <value>Failed to load image to local Docker daemon. stdout: {0}</value>
+    <value>Failed to load image to local Docker daemon. stdout: {0}.</value>
```

Comment:
if stdout is multiline, then period is not needed. 
stdout should have a period if formatted correctly.